### PR TITLE
Add PrivateHD release groups

### PR DESCRIPTION
**Pull request by ordinarygulp**
Add PrivateHD release groups

### CODE REVIEW

File: release-groups.txt [hunk 1]
```diff
@@ -3006,4 +3010,4 @@ Zuzuu
 ZX
 ZZGtv
 ~AA~
-ßsZ
\ No newline at end of file
+ßsZ
```
Comment (rednoah) [inline release-groups.txt:3013]: What's going on here? Please make sure the the commit doesn't contain any unnecessary changes.

Comment (ordinarygulp) [reply release-groups.txt:3013]: I have no idea. I didn't touch that. It must be something with my locale, because multiple browsers are making this change when I submit a commit.

Even the 'preview changes' function doesn't show that changed, until I commit. I guess you may add the groups since I'm unable to properly do so :( Tried to do the work for you so you wouldn't have to.

**Comment by rednoah** [comment]
I've added the missing groups.
